demisto_sdk/commands/common/tools.py

```diff
@@ -2455,6 +2456,22 @@ def should_alternate_field_by_item(content_item, id_set):
     return False
 
 
+def get_url_with_retries(url: str, retries: int, backoff_factor: int = 1, **kwargs):
+    kwargs['stream'] = True
+    session = requests.Session()
+    exception = Exception()
+    for _ in range(retries):
+        response = session.get(url, **kwargs)
+        try:
+            response.raise_for_status()
+        except Exception as error:
```

Comment:
```suggestion
        except HTTPError as error:
```

(might require importing it)